.docker/images/nginx/tests/src/BlocksTest.php

```diff
@@ -7,17 +7,15 @@
 /**
  * Ensure that the blocks are respected by nginx.
  */
-class BlocksTest extends TestCase
-{
+class BlocksTest extends TestCase {
 
   /**
    * Aggressive bots.
    *
    * @return array
    *   List of user agents.
    */
-  public function provideAggressiveAgents()
```

Comment:
here and below - updated data providers names.

it is a best practice to name data providers with `provider` prefix and the name of the test without `test` prefix.